docs/root/start/building.rst

```diff
@@ -33,7 +33,9 @@ as the new tcmalloc code is not guaranteed to compile with lower versions of Cla
 Windows Requirements
 --------------------
 
-Envoy has been ported and is now expected to work on Windows.
+Envoy now suports Windows as a target platform. The requirements below only apply if you want to build the Windows
+native executable. If you want to build the Linux version of Envoy on Windows either with WSL or Linux containers
+please see the Linux requirements.
```

Comment:
cool - perhaps make "linux requirements" a link and add "...above" at the end

Reply:
adding a link is a bit awkward because it doesn't change the link or even where you are on the page

Reply:
can we at least add "above" then - im just concerned that if you are on windows and are trying to follow then it helps to spell out that you just need to rewind reading

Reply:
added in f99c48964cd313ab7481a126bb11c0b0a453f7e3